SystemMicrocontroller_Sensors/Core/Inc/lcd20x4.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * lcd20x4.h
+ *
```

Comment:
Macros only used in driver should be define inside the C file. Macros used outside the driver may stay in the header file.